### PR TITLE
second commit

### PR DESCRIPTION
**Pull request by Denvar94**
Possible changes

### CODE REVIEW

File: countdown.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/ python
+from datetime import timedelta
```
Comment (famanson) [inline countdown.py]: unused import?

File: countdown.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+from datetime import datetime
+from datetime import date
```
Comment (famanson) [inline countdown.py]: we can also do away with this import if you use `datetime.now()` instead of `date.today()`

File: countdown.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+
+def run():
+    parser = OptionParser()
+    parser.add_option("-d", "--deadline", dest="date",
+                      help="Specify the deadline in ISO format: yyyy-mm-dd", metavar="DEADLINE")
+    parser.add_option("-e", "--event", dest="event", 
+                      help="Name of the deadline event",metavar="EVENT")
+    (options, args) = parser.parse_args()
+    if options.date is not None:
```
Comment (famanson) [inline countdown.py]: it should be sufficient to use `if not options.date`

File: countdown.py [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+        if options.event is not None:
```
Comment (famanson) [inline countdown.py]: same as above

File: countdown.py (hunk 1 above)
Comment (Denvar94) [inline countdown.py]: Yes, my original code used timtedelta,  I was in a rush and forgot to remove.

Comment (famanson) [inline countdown.py]: It's ok. Unless there's any objection, can you change the code according to my comments too?

**Pull request merged**